providers/component/src/webapp/WEB-INF/unboundid-ldap.xml

```diff
@@ -144,6 +144,30 @@
 			<value>false</value>
 		</property -->
 
+		<!-- Specifies the length of time in milliseconds between periodic background
+			health checks against the available connections in this pool.
+			Requires healthCheckMappings's invokeForBackgroundChecks to be set to true
+			Defaults to 60000L -->
+		<!-- property name="healthCheckIntervalMillis">
+			<value>60000</value>
+		</property -->
+
+		<!-- Provides mapping to create an LDAP connection pool health check implementation
+			that may be used to check the health of the associated server by verifying that
+			a specified entry can be retrieved in an acceptable period of time
+			Defaults to null (invokes the Unboundid defaut implementation, which may not do any checks) -->
```

Comment:
this looks good. can you be more definitive with this sentence ("may not")? what happens if this is unset?

Reply:
done.  I rechecked the LDAPConnectionPoolHealthCheck class in UnboundId's 4.0.12 source code and and the default health mostly does

`// No processing is performed in this default implementation.`

and the only method that does something, ensureConnectionValidAfterException(), only throws an error if the connection is bad.